pangeanetwork.py

```diff
@@ -266,6 +266,32 @@ def members():
     # TODO: request email confirmation if email was given
     return Response(json.dumps(result, default=str), mimetype='application/json')
 
+@app.route('/member/<memberid>', methods=['GET'])
+def member(memberid):
+  '''
+  GET: queries the database for all the users and returns the member that matches the id sent as a parameter
+    parameter:
+      id: the id of the member
+  '''
+  memberid = int(memberid)
+
+  if(request.method == 'GET'):
+    data = []
+    users = User.query.all()
```

Comment:
same as the other one- we should be able to filter the query here too to make it a little bit quicker. Otherwise looks great! 👍